packages/shared/src/limit/utils.ts

```diff
@@ -1,38 +1,32 @@
-import { Version } from '@/modules/core/domain/commits/types'
-import { GetWorkspaceLimits } from '@speckle/shared/dist/commonjs/authz/domain/workspaces/operations'
 import dayjs from 'dayjs'
+import { GetWorkspaceLimits } from '../authz/domain/workspaces/operations.js'
+import { PersonalProjectsLimits } from '../authz/index.js'
 
-export const PersonalProjectsLimits: {
-  versionHistory: { value: number; unit: 'week' }
-} = {
-  versionHistory: {
-    value: 1,
-    unit: 'week'
-  }
-}
-
-export const getLimitedReferencedObjectFactory =
+export const hidePropertyIfOutOfLimitFactory =
```

Comment:
why is this not a simple boolean check? you pass in something, that has a createdAt and you get back a bool. would be a lot more readable...